appveyor.yml

```diff
@@ -3,7 +3,6 @@ environment:
     - nodejs_version: '8'
     - nodejs_version: '10'
     - nodejs_version: '12'
-    - nodejs_version: '14'
```

Comment:
This might be unnecessary to be changed in this PR.

Reply:
Just came into my mind, this file should be removed, since we are using gihub to do CI/CD.